.github/workflows/main.yml

```
@@ -35,11 +35,11 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 18.x]
```

Only need to run lint once.
```suggestion
        node-version: [18.x]
```